index.d.ts

```diff
@@ -51,6 +57,19 @@ camelCase(['foo', 'bar']);
 
 camelCase(['__foo__', '--bar'], {pascalCase: true});
 //=> 'FooBar'
+
+camelCase('lorem-ipsum', {locale: 'en-US'});
+//=> 'loremIpsum'
+
+camelCase('lorem-ipsum', {locale: 'tr-TR'});
+//=> 'loremİpsum'
+
+camelCase('lorem-ipsum', {locale: ['en-US', 'en-GB]});
+//=> 'loremIpsum'
```

Comment:
The readme and index.d.ts should be fully in sync.

Reply:
I am a little confused about this. I interpreted your next comment as "Keep a single example in `index.d.ts` and move others to `readme.md` file", which breaks sync.